src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Tree/Dependencies/DependenciesGraphProvider.cs

```diff
@@ -522,15 +526,22 @@ Graph IGraphProvider.Schema
                                               string projectPath,
                                               IProjectDependenciesSubTreeProvider subTreeProvider,
                                               GraphNode parentNode, 
-                                              IEnumerable<IDependencyNode> nodes)
+                                              IEnumerable<IDependencyNode> nodes,
```

Comment:
So this method is only called to construct the nodes for search results and not for the regular tree?